client/src/components/PitchDoc/ClaimPitchModal.tsx

```diff
@@ -0,0 +1,291 @@
+      setOpenTeams(resp.data.result);
+    }
+  }, [pitch._id]);
+
+  const getUser = useCallback(async () => {
+    const contributors: { [name: string]: string } = {};
+    for (const userId of pitch.assignmentContributors) {
+      const res = await loadUser(userId);
+      if (!isError(res)) {
+        contributors[
+          `${res.data.result.firstName} ${res.data.result.lastName}`
+        ] = temp_pfp;
+      }
+    }
+    setPitchContributors(contributors);
+    if (pitch.pitchAuthor) {
+      const res = await loadUser(pitch.pitchAuthor);
+      if (!isError(res)) {
+        setPitchAuthor(
+          `${res.data.result.firstName} ${res.data.result.lastName}`,
+        );
+      }
+    }
+    if (pitch.approvedBy) {
+      const res = await loadUser(pitch.approvedBy);
+      if (!isError(res)) {
+        setApprovedBy(
+          `${res.data.result.firstName} ${res.data.result.lastName}`,
+        );
+      }
+    }
+  }, [
+    pitch.assignmentContributors,
+    pitch.pitchAuthor,
+    pitch.approvedBy,
+    temp_pfp,
+  ]);
+
+  useEffect(() => {
+    getTeams();
+    setSelectedTeams([]);
+    getUser();
+    setPitchLink('');
+  }, [firstOpen, getTeams, getUser]);
+
+  return (
+    <>
+      <Modal
+        className="claim-pitch-modal"
+        onClose={() => {
+          setFirstOpen(false);
+        }}
+        onOpen={() => {
+          setFirstOpen(true);
+        }}
+        open={firstOpen}
+        trigger={<PitchCard pitch={pitch} openTeams={openTeams}></PitchCard>}
+        closeIcon
+      >
+        <Modal.Actions>
+          <div className="claim-pitches-wrapper">
+            <div className="pitch-info">
+              <div className="topics-wrapper">
+                <Label.Group style={{ marginBottom: 10 }} circular>
+                  {pitch.topics.map((topic, idx) => (
+                    <>
+                      <Label
+                        style={{
+                          backgroundColor:
+                            interestsButtons[enumToInterestButtons[topic]],
+                        }}
+                        key={idx}
+                      >
+                        {enumToInterestButtons[topic]}
+                      </Label>
+                    </>
+                  ))}
+                </Label.Group>
+              </div>
+              <div className="pitch-name"> {pitch.name} </div>
+              <span className="submitted-reviewed-by">
+                Submitted By: {pitchAuthor}
+              </span>
+              <span className="submitted-reviewed-by">
+                Reviewed By: {approvedBy}
+              </span>
+              <div className="section-description">
+                Here lies a two sentence summary of pitch. It will be two
+                sentences and no more.
+              </div>
+            </div>
+            <div className="pitch-link">
+              <div className="pitch-link-title">Link to Pitch:</div>
+              <Input
```

Comment:
Change this to just display the link as an <a href="..."> - we don't need the user to be able to input anything here